tests/model/test_errata_model.py

```diff
@@ -1,19 +1,29 @@
 import os
+import json
 
 import pytest
 
-from pubtools.pulplib import ErratumUnit
+from pubtools.pulplib import ErratumUnit, Unit
 
 from pubtools.pulplib import InvalidDataException
 
 
+def test_container_list(data_path):
+    """Provide invalid structure object to container_list attribute."""
```

Comment:
That's not what the test is doing, is it?  It doesn't seem like the test data file named `sample-erratum.json` should contain invalid data.

Reply:
True, I copy pasted it from the existing one

Reply:
With the latest change I think you updated the doc string on the wrong test.  Now each test's doc string says the *opposite* of what it's testing...